app/views/partials/theme/simple/layout/header.liquid

```diff
@@ -17,6 +17,11 @@
       <a href="/groups" class="p-3 hover:text-gray-900">{{ 'app.groups.list' | t }}</a>
 
       {% if context.current_user.id %}
+        <a href="/inbox" class="flex mr-5 hover:text-gray-900"  id='inbox-notifications' data-current-user-id="{{ profile.id }}" data-current-user-slug="{{ profile.slug }}">
```

Comment:
@ignacy Few lines below there is already `Inbox` position in header. Replace it or remove it.